client/consensus/aura/src/lib.rs

```diff
@@ -148,7 +148,7 @@ pub fn start_aura<B, C, SC, E, I, P, SO, CAW, Error, H>(
 	force_authoring: bool,
 	keystore: KeyStorePtr,
 	can_author_with: CAW,
-) -> Result<impl futures01::Future<Item = (), Error = ()>, sp_consensus::Error> where
+) -> Result<impl futures::Future<Output = ()>, sp_consensus::Error> where
```

Comment:
The style was `futures01::` in order to make it obvious that we have a futures 0.1. Now that there's only a single future type, we should do some nit-picky styling.

```suggestion
) -> Result<impl Future<Output = ()>, sp_consensus::Error> where
```